edx-platform/pearson-certiport-theme/lms/templates/header/user_dropdown.html

```diff
@@ -3,36 +3,35 @@
 <%namespace name='static' file='static_content.html'/>
```

Comment:
Check the production environment to verify that these settings apply to the Certiport site. @sergivalero20

Reply:
@Squirrel18  It only applies to courses-pathways site.

Reply:
So, I'll remove it.